### DISCUSSION TITLE
Importing from google translated website

### DISCUSSION THREAD

**Discussion by siz-** [mealie-recipes/mealie#3865] [General]
Hey,

I just started using Mealie and its great. I would like to able to bulk import google translated recipes. However, I can't get it to work. For example from BBC goodfood: 

https://www-bbcgoodfood-com.translate.goog/recipes/zingy-chicken-stir-fry?_x_tr_sl=en&_x_tr_tl=da&_x_tr_hl=en-US&_x_tr_pto=wapp&_x_tr_hist=true

Mealie imports the original english version, and not the translated version. I think there only needs to be added a delay, as the webpage needs a second to translate the page? 

Is there any way to achieve this? 👍

**Comment by Kuchenpirat** [comment]
Hi, 
Thanks for the compliments :)

I don't think it is possible to use the translated results with our current implementation. Mealie does not scrape the visible data but relies on a [recipe schema ](https://schema.org/Recipe) to be present which will not get translated by google translate.
